core/src/main/java/com/linecorp/armeria/client/retry/RetryingClient.java

```diff
@@ -85,6 +87,13 @@ public static RetryingClientBuilder builder(RetryRuleWithContent<HttpResponse> r
         return new RetryingClientBuilder(retryRuleWithContent, maxContentLength);
     }
 
+    /**
+     * Returns a new {@link RetryingClientBuilder} with the specified {@link RetryConfigMapping}.
+     */
+    public static RetryingClientBuilder builderWithMapping(RetryConfigMapping<HttpResponse> mapping) {
```

Comment:
`builder()` or `builderWithMapping()`?

Reply:
builderWithMapping() name is used instead of a normal overload for builder(), because the overload is ambiguous with an existing signature builder(RetryRule). Java does not understand that lambdas following the functional interface for RetryRule and RetryConfigMapping are of different types, since both have two arguments, although the arguments are of different types.